test/db/analysis/vars

```diff
@@ -1112,125 +1059,3 @@ EXPECT=<<EOF
 |           0x8000054e      ret   
 EOF
 RUN
-
-NAME=Function va_args arguments from DWARF
-FILE=bins/elf/float_ex1/float_ex1_hightec
```

Comment:
Why you removed this and other hightec test?

Reply:
https://github.com/rizinorg/rizin/blob/c649ba025440cbed5f14161cc8a3f7ae6c421384/test/db/analysis/vars#L866-L889

These two tests were repeated. The other one is somewhat useless (there should be no need for another 'pdf' test here, I think we only need to test function signatures here?) ), so it was merged with another hightec test.

Reply:
Sounds good.

Reply:
Right, we should test funciton signatures and variables/arguments (`afs` and `afv` commands)